packages/kbn-openapi-generator/src/template_service/templates/zod_schema_item.handlebars

```diff
@@ -80,7 +80,11 @@ z.unknown()
       * {{{description}}} 
       */
       {{/if}}
-      {{@key}}: {{> zod_schema_item requiredBool=(includes ../required @key)}},
+      {{#if (startsWithSpecialChar @key)}}
```

Comment:
Our `@timestamp` field broke the code generator.

There is a separate PR for this fix: https://github.com/elastic/kibana/pull/172049